src/main/kotlin/lottery/domain/LotteryMatcher.kt

```diff
@@ -1,21 +1,31 @@
 package lottery.domain
 
-class LotteryMatcher(private val winnerLottery: WinnerLottery, private val createdLotteries: List<Lottery>) {
-    fun match(): RankCounts {
-        val rankCounts = RankCounts()
+class LotteryMatcher(
+    private val winnerLottery: WinnerLottery,
+    private val createdLotteries: List<Lottery>
+) {
+    fun match(bonusBall: BonusBall): RankCounts {
+        var rankCounts = RankCounts()
 
         createdLotteries.map {
```

Comment:
```suggestion
        createdLotteries.forEach {
```
map으로 생성된 값을 쓰고 있지 않아 보이네요 :)